Library/Homebrew/cli/parser.rb

```diff
@@ -296,7 +296,9 @@ def parse_remaining(argv, ignore_invalid_options: false)
         [remaining, non_options]
       end
 
-      sig { params(argv: T::Array[String], ignore_invalid_options: T::Boolean).returns(Args) }
+      # @return [Args] The actual return type is `Args`, but since `Args` uses `method_missing` to handle options, the
+      #   `sig` annotates this as returning `T.untyped`` to avoid spurious type errors.
+      sig { params(argv: T::Array[String], ignore_invalid_options: T::Boolean).returns(T.untyped) }
```

Comment:
Note that there are ~20 other files in `cmd/` that we should be able to flip to `typed: true` with this change, which I can take on in a follow-up PR.